app/src/main/res/layout/view_error_banner.xml

```diff
@@ -40,16 +40,16 @@
         android:id="@+id/error_action"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginEnd="16dp"
+        android:layout_marginEnd="12dp"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toStartOf="@id/error_close"
         app:layout_constraintTop_toTopOf="parent" />
 
     <androidx.appcompat.widget.AppCompatImageButton
         android:id="@+id/error_close"
-        android:layout_width="16dp"
-        android:layout_height="16dp"
-        android:layout_marginEnd="12dp"
+        android:layout_width="48dp"
```

Comment:
Can this be reduced to 32dp x 32dp - this gives the text more room on smaller screens while still allowing the button to be easily clicked (32x32dp is a common clickable size)

Reply:
I made this change on the main branch